apps/condo/domains/resident/schema/RegisterServiceConsumerService.js

```diff
@@ -56,35 +60,33 @@ const RegisterServiceConsumerService = new GQLCustomSchema('RegisterServiceConsu
 
                 if (!tin) { throw new Error(`${REQUIRED_NO_VALUE_ERROR}tin] tin null or empty: ${tin}`) }
 
-                // Todo (DOMA-1604) add filtering by tin
-                // Right now we don't actaully use tin for getting an organization, but get organization from resident. Reasons:
-                // 1. We can't optimally filter on tin: (DOMA-1604)
-                // 2. Right now it's okay to get organiation from resident. We would need to get organization by tin when we add new Acquiring
-                // const [ organization ] = await Organization.getAll(context, { meta_in: { tin: tin } })
-                const [ organization ] = await Organization.getAll(context, { id: get(resident, ['organization', 'id']) })
+                // GraphQL from Keystone does not supports querying of database fields of type JSON.
+                const { keystone } = await getSchemaCtx('Organization')
```

Comment:
Better to move this to some function, as it will be refactored after TIN will have its own field in the Organization model